xio-core/src/main/java/com/xjeffrose/xio/http/ClientChannelInitializer.java

```diff
@@ -64,7 +63,6 @@ protected void initChannel(Channel channel) throws Exception {
     }
     channel
         .pipeline()
-        .addLast("idle handler", new XioIdleDisconnectHandler(60, 60, 60))
```

Comment:
i think client is meant to be used not just as a proxy client... i wonder if the disconnect handler should be removed by the proxy client factory instead

Reply:
... or better yet a configurable timeout

Reply:
yeah making update, good idea

Reply:
I will add tests as well